docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
           echo '.venv exists and is not empty'; 
         else 
           echo '.venv does not exist or is empty'; 
-          python3.10 -m venv .venv; 
+          python3.10 -m venv .venv;
```

Comment:
I think this is the only way to make this service work as intended as Python 3.11 is unavailable in the parent Docker image we start from.